src/parser/ParsedQuery.h

```diff
@@ -408,18 +410,21 @@ class ParsedQuery {
   }
 
   void expandPrefixes();
+  // unused function ? not implemented
   void parseAliases();
```

Comment:
Yes, please remove it.
In the meantime we parse Aliases using a different mechanism.